src/storage/src/render/sinks.rs

```diff
@@ -164,6 +165,18 @@ where
         collection.map(|row| (None, row))
     };
 
+    fn warn_on_dups<T>(v: &[T], sink_id: GlobalId, from_id: GlobalId) {
+        if v.len() > 1 {
+            warn!(
```

Comment:
could this be really noisy?

Reply:
additionally, it might be nice to document that this can only happen if using `NOT ENFORCED`?

Reply:
It's possible for this to be noisy iff there's a ton of conflicts... so far they're pretty rare but it's definitely possible. If needed we could definitely use some internal state to throttle these messages.